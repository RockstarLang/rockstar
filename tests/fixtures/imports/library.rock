Say "Importing library.rock"

Put 42 into Foo.

Bar takes X
Say "In function library.rock Bar(" plus X plus ")"
(end function)

Put 3.14159 into the pi,

Put 2.71828 into the e

If thy Name is nobody
Say thy name!

Say "Imported library.rock"
