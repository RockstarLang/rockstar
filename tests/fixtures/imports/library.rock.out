Importing library.rock

Imported library.rock
